card/lib/card/migration.rb

```diff
@@ -62,12 +62,10 @@ def contentedly
     Card::Cache.reset_all
     Cardio.schema_mode "" do
       Card::Auth.as_bot do
-        ActiveRecord::Base.transaction do
```

Comment:
interesting.  don't migrations normally do this?  Was this added in because of the schema change?